### DISCUSSION TITLE
Use Model without specifying columns of table?

### DISCUSSION THREAD

**Discussion by Uttam-EWW** [adonisjs/core#3438] [Help]
So basically, I want to use model object for querying but without specifying all columns of table.
I tried this approach but I couldn't execute it. I below errors.

![image](https://user-images.githubusercontent.com/86663119/147039704-faccc01b-7f53-416b-a7dc-1ac57b664789.png)

So is there any way to use model without specifying columns?
Thanks.

**Comment by Barbapapazes** [comment]
Hello, can you provide more informations and a snippet of code?

**Reply by Uttam-EWW** [reply]
```export default class Company extends BaseModel {}```
Let's say above ```Company``` is my model, I'm not specifying any column names as properties. So my Question is, is there a way to use all ```BaseModel``` methods ?
I tried but giot above errors.

**Reply by Barbapapazes** [reply]
Yep, you can use the [query builder](https://docs.adonisjs.com/reference/database/query-builder)!

``` ts
import Database from '@ioc:Adonis/Lucid/Database'

Database.from('compagny').select('*').first()
```

I think the code snippet above can help you!

**Reply by Uttam-EWW** [reply]
I know about this way, but can not access other model methods, like ```firstOrCreate```, ```updateOrCreate```, etc.

**Reply by Barbapapazes** [reply]
And how a model can have no columns but data in database? It seems to be a strange case!

**Reply by Uttam-EWW** [reply]
of course there is columns in database, but just don't want write it down in model.

**Reply by carlmathisen** [reply]
@Uttam-EWW There's a lot of useful "magic" in Lucid models, which can only work optimally if the model is configured properly. If you don't want to configure the models, then you're better off just doing a regular query with the `Database` query builder like @Barbapapazes suggested.
